client/my-sites/plans/current-plan/jetpack-checklist/footer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { untrailingslashit } from 'lib/route';
+import Button from 'components/button';
+import Card from 'components/card';
+
+class JetpackChecklistFooter extends PureComponent {
+	handleWPAdminLink = () => {
+		this.props.recordTracksEvent( 'calypso_checklist_wpadmin' );
+	};
+
+	render() {
+		const { translate, wpAdminUrl } = this.props;
+		return (
+			<Card compact className="jetpack-checklist__footer">
+				<p>{ translate( 'Return to your self-hosted WordPress dashboard.' ) }</p>
+				<Button compact href={ wpAdminUrl } onClick={ this.handleWPAdminLink() }>
+					{ translate( 'Return to WP Admin' ) }
+				</Button>
+			</Card>
+		);
+	}
+}
+
+export default connect(
+	state => {
+		const site = getSelectedSite( state );
+
+		return {
+			wpAdminUrl: untrailingslashit( get( site, 'options.admin_url' ) ) + '/admin.php?page=jetpack',
```

Comment:
Any changes `options.admin_url` is empty?

Reply:
I implemented something similar in #32790, the admin_url was used without checks before my changes. 🤷‍♂ 

You might re-use the big from #32790 which goes to the "My Plan" page of the dashboard:

https://github.com/Automattic/wp-calypso/blob/ccfa051749717cb59ef2fdacac508ba24861286b/client/my-sites/sidebar/sidebar.jsx#L664-L671

Reply:
I like the idea of adding some additional safety:

- Try to build the admin URL.
- If we don't have it, set this property to `null`

We don't render the component at all if `adminUrl` is `null` 👍

Reply:
>We don't render the component at all if adminUrl is null

Done 👍